build.gradle.kts

```diff
@@ -40,6 +40,8 @@ dependencies {
     api(libs.archunit)
     testImplementation(kotlin("test"))
     testImplementation(libs.archunit.junit)
+    testImplementation("org.springframework.boot:spring-boot-starter-data-jpa:3.1.0")
```

Comment:
plz use [spring deps management](https://docs.spring.io/dependency-management-plugin/docs/current/reference/html/) for versioning. Also, now we use gradle version catlog. You should not specify the version number in this file

Reply:
okay